lib/dbi/session.h

```diff
@@ -28,8 +28,10 @@
 extern "C" {
 #endif
 
+#define OGS_DBI_NO_CHARGING_CHAR -1
```

Comment:
so the peer cannot send an all 0xff value meaning something different?
Or it's because ChargingChracteristics is 16bit?